lib/eslint.js

```diff
@@ -10,7 +10,7 @@ function detectEslint(dir) {
 
     return Promise.all([
         tryFiles(paths),
-        tryPackageJson(dir, 'eslintConfig'),
+        tryPackageJson(dir, ['eslint', 'eslintConfig', 'eslintIgnore']),
```

Comment:
I couldn't find in the docs mentioning the `eslint` prop: https://eslint.org/docs/user-guide/configuring
Could you point to a reference?

Reply:
Oh I see, you want to check if there's a `eslint` dependency. See my comment below.

Reply:
Perhaps we should analyze all other linters and do the same for every linter that does not require a `config` prop in package.json and just the dependency installed?

Reply:
> Perhaps we should analyze all other linters and do the same for every linter that does not require a `config` prop in package.json and just the dependency installed?

Agreed. I only updated eslint because I was already familiar with it